src/settings-ui/Settings.UI/Services/ActivationService.cs

```diff
@@ -40,11 +40,13 @@ public async Task ActivateAsync(object activationArgs)
 
                 // Do not repeat app initialization when the Window already has content,
                 // just ensure that the window is active
+#pragma warning disable WinUIEx1001 // The member will always be null.
```

Comment:
looks like this entire class is not used anywhere. We can delete it (follow-up PR)